test/fail_compilation/diag10862.d

```diff
@@ -1,4 +1,4 @@
-/*
+/* REQUIRED_ARGS: -d
```

Comment:
Also removed the change to  test/fail_compilation/fail_arrayop2.d  for the same reason